graph/db/graph_test.go

```diff
@@ -788,10 +788,14 @@ func assertEdgeInfoEqual(t *testing.T, e1 *models.ChannelEdgeInfo,
 	if !bytes.Equal(e1.NodeKey2Bytes[:], e2.NodeKey2Bytes[:]) {
 		t.Fatalf("nodekey2 doesn't match")
 	}
-	if !bytes.Equal(e1.BitcoinKey1Bytes[:], e2.BitcoinKey1Bytes[:]) {
```

Comment:
Is the commit message correct? `V2 channels use taproot where the node keys serve as the bitcoin keys.` Looks like we still have the bitcoin keys on v2 messages, right?